### ISSUE TITLE
can't 'zig build run' on windows

### ISSUE THREAD

**Issue by GmosNM** [ziglang/zig#16897]
### Zig Version

0.11,0

### Steps to Reproduce and Observed Behavior

zig build run

the output is:
Semantic Analysis [667] error(mingw): clang exited with code 1 and stderr: error: unknown target triple 'x86-unknown-windows-gnu', please use -triple or -arch

Semantic Analysis [1768] error(mingw): clang exited with code 1 and stderr: error: unknown target triple 'x86-unknown-windows-gnu', please use -triple or -arch

Semantic Analysis [4957] error(mingw): clang exited with code 1 and stderr: error: unknown target triple 'x86-unknown-windows-gnu', please use -triple or -arch

error: unable to generate DLL import .lib file for advapi32: ClangPreprocessorFailed

### Expected Behavior

build and run the program

**Comment by Vexu** [comment]
This looks similar to https://github.com/ziglang/zig-bootstrap/issues/144, how did you acquire your Zig binary?

**Comment by GmosNM** [comment]
@Vexu 
https://ziglang.org/download/

![image](https://github.com/ziglang/zig/assets/37900384/29dda1d2-2b09-4cd6-9212-fa46de4f117d)

**Comment by GmosNM** [comment]
```
PS D:\dev> cd o
PS D:\dev\o> ls
PS D:\dev\o> zig init-exe
info: Created build.zig
info: Created src\main.zig
info: Next, try `zig build --help` or `zig build run`
PS D:\dev\o> zig build run
Semantic Analysis [735] error(mingw): clang exited with code 1 and stderr: error: unknown target triple 'x86-unknown-windows-gnu', please use -triple or -arch

Semantic Analysis [1742] error(mingw): clang exited with code 1 and stderr: error: unknown target triple 'x86-unknown-windows-gnu', please use -triple or -arch

Semantic Analysis [4957] error(mingw): clang exited with code 1 and stderr: error: unknown target triple 'x86-unknown-windows-gnu', please use -triple or -arch

error: unable to generate DLL import .lib file for advapi32: ClangPreprocessorFailed
PS D:\dev\o>
```

**Comment by GmosNM** [comment]
fixed it
```
PS D:\dev\o> zig build run
Hello, world!
PS D:\dev\o>
```

i downloaded [ig-windows-x86_64-0.11.0.zip](https://ziglang.org/download/0.11.0/zig-windows-x86_64-0.11.0.zip)
i was using [ig-windows-x86_64-0.11.0.zip](https://ziglang.org/download/0.11.0/zig-windows-x86_64-0.11.0.zip)

**Comment by gphilipp** [comment]
I have the same problem, and I don't understand how you fixed it as your previous comments points to the same packages 🤔

**Comment by tim-tim707** [comment]
Same issue, fixed using `zig-windows-x86_64-0.11.0` instead of `zig-windows-x86-0.11.0` (`x86_64` instead of `x86`)
